src/pages/explore/Explore.jsx

```diff
@@ -1,9 +1,38 @@
-import React from 'react'
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import VideoCard from '../../components/card/VideoCard';
+import CategoryChips from '../../components/chips/CategoryChips';
+import { getVideoList } from '../../features/video/videoSlice'; 
+import { getCategoryList } from '../../features/category/categorySlice';
+
+import '../../components/chips/CategoryChips.css';
+import './Explore.css';
 
 export function Explore() {
-  return (
-    <div>
-      explore page
+
+
+  const { videos } = useSelector((store)=>store.videoList);
+  const { categories } = useSelector((store)=>store.categoryList);
+
+  const dispatch = useDispatch();
+  console.log( videos);
+  
+  useEffect(()=>{
+    dispatch(getVideoList());
+    dispatch(getCategoryList());
+  },[])
```

Comment:
Didn't you have to mention dispatch as a dependency in the useEffect because when I was trying to do so a warning used to pop up related to adding dispatch in the dependency array?

Reply:
No, i haven't got any warnings ,if its an issue then we can discuss